crawl-ref/source/webserver/game_data/static/cell_renderer.js

```diff
@@ -472,8 +520,6 @@ function ($, view_data, main, tileinfo_player, icons, dngn, enums,
             }
             this.ctx.fillStyle = fg_term_colours[col.fg];
             this.ctx.font = prefix + this.glyph_mode_font_name();
-            this.ctx.textAlign = "center";
```

Comment:
The textAlign should remain center, since there isn't a robust programmatic way to check if a font is monospaced (people say compare I to M but that's adhoc) and using a non-mono font, while not supported officially, should at least have a chance of looking nice.